packages/core/src/cli/add-offical-connectors.ts

```diff
@@ -0,0 +1,16 @@
+import 'module-alias/register';
```

Comment:
The file name is incorrect.

Reply:
And I think you have this file in the base branch already?

Reply:
Rebased.